sql/xsql/src/main/scala/org/apache/spark/sql/xsql/manager/MysqlManager.scala

```diff
@@ -104,6 +104,21 @@ private[xsql] class MysqlManager(conf: SparkConf) extends DataSourceManager with
     }
   }
 
+  /**
+   * Cache special properties for mysql dataSource
+   */
+  private def cacheSpecialProperties(
+      dsName: String,
+      dbName: String,
+      tbName: String): Unit = {
+    val tablePartitionsMap = partitionsMap.get(dbName)
+    if (tablePartitionsMap != None) {
+      tablePartitionsMap.get.get(tbName).foreach { m =>
+        specialProperties += ((s"${dsName}.${dbName}.${tbName}", m))
+      }
+    }
```

Comment:
```
  Option(partitionsMap(dbName)).map{ tablePartitionsMap =>
    Option(tablePartitionsMap(tbName)).foreach { m =>
      specialProperties += ((s"${dsName}.${dbName}.${tbName}", m))
    }
  }
```

Reply:
Thanks for your advice， but the code can't pass.

Reply:
LGTM